data/stk_config.xml

```diff
@@ -281,40 +280,7 @@
        ============================ -->
   <general-kart-defaults>
 
-    <!-- Skidding: increase: multiplicative increase of skidding factor in each frame.
```

Comment:
This "Skidding" comments is relatade to no xml tag and is exactly the same in another part of the file.